scripts/README.md

```diff
@@ -45,50 +56,50 @@ To withdraw funds, all withdrawals have to be specified in a file with the follo
     }
 ]
 ```
+
 The script can automatically determine the amount, instead of having to specify it on the file.
 This is achieved by adding the flag `--allTokens` to the withraw command. This is possible in any of the following commands.
 
 Withdrawing is a two-step process: first a withdraw must be requested on the exchange, then the withdraw is executed and at the same time the funds are sent back to the master Safe.
 
 ```js
-truffle exec scripts/withdraw.js --requestWithdraw --masterSafe=0xd9395aeE9141a3Efeb6d16057c8f67fBE296734c --withdrawals="./data/depositList.json" --network=rinkeby
+truffle exec scripts/withdraw.js --requestWithdraw --masterSafe=$MASTER_SAFE --withdrawals="./data/depositList.json" --network=$NETWORK_NAME
 ```
 
 ```js
-truffle exec scripts/withdraw.js --withdraw --transferBackToMaster --masterSafe=0xd9395aeE9141a3Efeb6d16057c8f67fBE296734c --withdrawals="./data/depositList.json" --network=rinkeby
+truffle exec scripts/withdraw.js --withdraw --transferBackToMaster --masterSafe=$MASTER_SAFE --withdrawals="./data/depositList.json" --network=$NETWORK_NAME
 ```
 
 The latter instruction can be split into two independent units, if needed: withdrawing from the exchange to the traders and transferring funds from the traders to the master Safe.
 
 ```js
-truffle exec scripts/withdraw.js --withdraw --masterSafe=0xd9395aeE9141a3Efeb6d16057c8f67fBE296734c --withdrawalsFromDepositFile="./data/depositList.json" --network=rinkeby 
+truffle exec scripts/withdraw.js --withdraw --masterSafe=$MASTER_SAFE --withdrawalsFromDepositFile="./data/depositList.json" --network=$NETWORK_NAME
 ```
 
 ```js
-truffle exec scripts/withdraw.js --transferBackToMaster --masterSafe=0xd9395aeE9141a3Efeb6d16057c8f67fBE296734c --withdrawalsFromDepositFile="./data/depositList.json" --network=rinkeby 
+truffle exec scripts/withdraw.js --transferBackToMaster --masterSafe=$MASTER_SAFE --withdrawalsFromDepositFile="./data/depositList.json" --network=$NETWORK_NAME
 ```
 
 ### Full Cycle Test
 
-To begin, you must have an existing "Master Safe" created through the interface (here)[https://rinkeby.gnosis-safe.io] for convienience, add an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the  "Proposer" account, who will be responsible for submitting the transaction from the terminal and ensure the safe requires at least two transactions. Note that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
+To begin, you must have an existing "Master Safe" created through the interface (here)[https://rinkeby.gnosis-safe.io] for convienience, add an additional owner `0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1`, known as the "Proposer" account, who will be responsible for submitting the transaction from the terminal and ensure the safe requires at least two transactions. Note that the mnemonic phrase for this "Proposer" account is stored in plain text within this project.
```

Comment:
Maybe we want to stress even further that this address must not be the single owner of the multisig, otherwise anyone could steal all the funds stored there.